### ISSUE TITLE
 Ability to get data of multiple institutes

### ISSUE THREAD

**Issue by lablnet** [riuisb/web_resource_cli#5]
**Ability to get data of multiple institutes**

It's a bit tricky, but i have the following approach in my mind.
We create, `json` file that will define structure of university data what we want to get.
`url`, `field` blah blah , so demonstration here is look of sample 
```json
{
  "riphah": {
    "url" : "https://riphah.edu.pk",
    "facilities": {
      "computing": {
        "end": "/faculty/ict-computing",
        "members" : {
          "end": "/faculty-members"
        }
      }
    }
  }
}
```

and every data will be saved in `json` file, on university name we create directory, and all data will be under `data` directry,

@Usman-Naeem 
what do you think?

**Comment by muhammadusmannaeem** [comment]
Yes that might be just what I was looking for, using that we can easily parse data from multiple websites. And all we need to do is get one time data in this json file. Thanks
